webapp/components/Registration/RegistrationSlideInvite.vue

```diff
@@ -124,18 +122,11 @@ export default {
     async handleInputValid() {
       this.sendValidation()
     },
-    isVariablesRequested(variables) {
-      return (
-        this.sliderData.sliders[this.sliderIndex].data.request &&
-        this.sliderData.sliders[this.sliderIndex].data.request.variables &&
-        this.sliderData.sliders[this.sliderIndex].data.request.variables.code === variables.code
-      )
-    },
     async handleSubmitVerify() {
       const { inviteCode } = this.sliderData.collectedInputData
       const variables = { code: inviteCode }
 
-      if (!this.isVariablesRequested(variables) && !this.dbRequestInProgress) {
+      if (!this.dbRequestInProgress) {
```

Comment:
Hmm. I think the purpose of `isVariablesRequested()` was to check if the same request was made before. Now it could happen that a request to check if a code is valid is not made, because _another_ code is being checked.